libvmi/driver/kvm/README.md

```diff
@@ -0,0 +1,93 @@
+# KVM Driver
+
+## Overview
+
+The actual KVM driver contains 2 implementations
+
+- KVMi: the new [KVM virtual machine introspection API](https://static.sched.com/hosted_files/kvmforum2019/f6/Advanced%20VMI%20on%20KVM%3A%20A%20progress%20Report.pdf)
+- Legacy: the legacy KVM driver for LibVMI, using either `GDB` or the `fast-memaccess` patches available in `libvmi/tools/qemu-kvm-patch`
+
+## LibVMI API Implementation
+
+This section will give an implementation status of the LibVMI API on the new KVM driver.
+
+- [x] r/w physical memory
+- [ ] VCPU registers (x86 only)
```

Comment:
@Wenzel Does this file need updates? Example: It seems vcpu read/write is documented as unavailable, but I did see get/set implemented.

Reply:
Go catch, I should have checked the sub checkboxes here

Reply:
libkvmi supports setting general purpose registers only, for now. No special registers and no MSRs.

Reply:
Now I remember, I didn't check the boxes because of these restrictions.
I suppose I should add a sub-level of boxes on both read and write sections, and check them for general purpose registers, so it's more clear what is supported ATM.

Reply:
@Wenzel  yes please, thank you